hazelcast/src/main/java/com/hazelcast/internal/partition/InternalPartitionService.java

```diff
@@ -79,6 +79,11 @@
      */
     void memberRemoved(Member member);
 
+    /**
+     * Returns whether partition table is initialized or not.
+     */
+    boolean isPartitionStateInitialized();
```

Comment:
```suggestion
    boolean isPartitionTableInitialized();
```

Reply:
I'll remove this method, as it's not used apart from a test.